### ISSUE TITLE
[Acccessibility][Sourcing]: fdp-table navigation issues using keyboard

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9131]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
There are some navigation issues with regards to <fdp-table>

1. Reach using keyboard on a blank or empty table cell - Notice that up/ down arrow doesn’t work and only left / right arrow key works on this type of cell
2. Reach the table first time using keyboard (tab key press)- Now once the focus is on the first column of table - Notice that left / right arrow keys does not work to traverse to other column headers
3. Non interactive elements (text) do no show visible focus - so user is not aware which cell he / she is currently on but screen reader reads the cell values 
4.  Not able to easily skip the table cells : Jump between first and last cell in table not working: 
for windows refer: https://www.w3.org/WAI/ARIA/apg/example-index/grid/dataGrids.html 
for mac: fn + option + right arrow
5. Come out of table : On press of tab key inside table anywhere, the focus should come out of table and move to next interactive element

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 13.x.x
   "@fundamental-ngx/core": "0.35.4",
    "@fundamental-ngx/platform": "0.35.4",

#### If this is a bug, please provide steps for reproducing it.
replicable on ngx site

#### Please provide relevant source code if applicable.
These are reproducible on ngx website : https://sap.github.io/fundamental-ngx/#/platform/table 

#### Is there anything else we should know?
Please contact us for any further information. These issues are blocker for Deutsche Telecom customer for guided sourcing.

**Comment by mikerodonnell89** [comment]
@platon-rov assigning this to you since you have a better understanding of fdkFocusableGrid

**Comment by I543348** [comment]
This is fixed
